sklearnex/tests/test_patching.py

```diff
@@ -139,6 +140,9 @@ def test_standard_estimator_patching(caplog, dataframe, queue, dtype, estimator,
             ]:
                 pytest.skip(f"{estimator} does not support GPU queues")
 
+        if "NearestNeighbors" in estimator and "radius" in method:
+            pytest.skip(f"RadiusNeighbors estimator not implemented in sklearnex")
```

Comment:
I wouldn't turn off testing. On our side, sklearn provides functional support for dpnp/dpctl interop. Tests should not fail

Reply:
Thank you for the suggestion.  In this case, we discussed how we wanted to give the user an informative message when using radius_neighbors and radius_neighbors_graph when setting the sklearnex logger to debug, which should tell them that they are using sklearn functionality.  This is done here: https://github.com/icfaust/scikit-learn-intelex/blob/dev/n_jobs_refactor/sklearnex/neighbors/knn_unsupervised.py#L96 . In so doing, this will trigger the sklearnex testing fallback guard here: https://github.com/icfaust/scikit-learn-intelex/blob/dev/n_jobs_refactor/sklearnex/conftest.py#L52 Therefore causing the tests with radius_neighbors and radius_neighbors_graph to fail.  We could turn off the fallback guard, but then that would impact the entirety of the patching tests that this is a part of. @ethanglaser and I discussed and decided that this was the best approach. @samir-nasibli let me know what you think.